### PR TITLE
--current --skip-deps become critical, so we add them to the CI.

### PR DESCRIPTION
**Pull request by tanin47**
People are using `--current --skip-deps` intensely. We need to make sure they don't fail.

### CODE REVIEW

File: ci/continuous_integration/daemon/main.py [hunk 1]
```diff
@@ -221,12 +224,14 @@ def reset_and_run_testcase(testcase_id, category, release):
   process.call('git clean -d -f -f', cwd=CHROMIUM_SRC)
 
   version = prepare_binary_and_get_version(release)
-  update_auth_header()
-  return_code = run_testcase(testcase_id)
-  logs = read_logs()
 
-  stackdriver_logging.send_run(
-      testcase_id, category, version, release, return_code, logs)
+  for opts in TESTED_OPTS:
```
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: What happens here is:

1. Run `clusterfuzz reproduce 12345`
2. Then, run `clusterfuzz reproduce 12345 --current --skip-deps` immediately afterward. The build time should be zero here.

File: ci/continuous_integration/daemon/main.py [hunk 2]
```diff
@@ -36,6 +36,9 @@
     error.MinimizationNotFinishedError.EXIT_CODE
 ])
 
+# The options that will be tested on the CI.
+TESTED_OPTS = ['', '--current --skip-deps']
```
Comment (inferno-chromium) [inline ci/continuous_integration/daemon/main.py]: s/OPTS/OPTIONS everywhere please.

Comment (inferno-chromium) [inline ci/continuous_integration/daemon/main.py]: s/TESTED/TEST sounds better.

File: ci/continuous_integration/tests/daemon/main_test.py [hunk 3]
```diff
@@ -94,11 +94,12 @@ def test_not_save_testcase_id(self):
...
         error.MinimizationNotFinishedError.EXIT_CODE, None)
     self.assertEqual(
-        error.MinimizationNotFinishedError.EXIT_CODE, main.run_testcase(1234))
+        error.MinimizationNotFinishedError.EXIT_CODE,
+        main.run_testcase(1234, ''))
 
     self.assert_exact_calls(self.mock.call, [
         mock.call(
-            '/python-daemon-data/clusterfuzz reproduce 1234',
+            '/python-daemon-data/clusterfuzz reproduce 1234 ',
```
Comment (inferno-chromium) [inline ci/continuous_integration/tests/daemon/main_test.py]: nit: we can fix this to not have whitespace at end.

**Pull request merged**